### ISSUE TITLE
Option to hide "_build_tuple('_winapi'): could not locate source" messages

### ISSUE THREAD

**Issue by mpdehaan** [dw/mitogen#444]
A lot of the _underscore speedup or optional modules generate warnings on screen, that will result in filing tickets about the applications, when they are really safe to ignore.

EXAMPLE:

_build_tuple('_winapi'): could not locate source

Having some facility to either manually provide a list of these modules to ignore to the system, or globally disable all of these messages, is something I'd really need to have to use this.

**Comment by dw** [comment]
So far when this comes up I've just been lowering the log levels involved. Really all this stuff is basically just debug output. Ideally 'mitogen.importer' would be a separate component that could have ERROR lever set by default, so these things would be all filtered entirely. It's actually easy to fix, but it requires figuring out which logger categories are most useful (Presently, debug output is overwhelming for pretty much everyone including me)

**Comment by mpdehaan** [comment]
Ah, so I really should blacklist those... good point... (I have yet to dig in to how to do that)...

**Comment by dw** [comment]
it's not documented yet!   `mitogen.master.Router` is the temporary home for a `.responder` attribute, which is a `mitogen.master.ModuleResponder`. It has a `router.responder.blacklist_prefix(...)` function that lets you set string prefixes for functions the target should never round-trip for. Before starting any context, just call e.g. `blacklist_prefix('_winapi')` and the problem should go away.

Note this applies to round-trips the target should never make. The log message you hit is due to module dependency scanning in the master (i.e. the error may be due to some import of some import of some import for a legit package we always use). In that case, the log level lowering fix committed above is the only solution.

IIRC there is a ticket open elsewhere (and if not, this can be it) to automatically handle speedup modules. The master knows when the dep scanner hits a native module -- it just doesn't tell the target not to bother trying to load it, because it will be refused. A list of native modules just needs to be made an additional field in the `LOAD_MODULE` message

**Comment by dw** [comment]
Closing this in favour of the older 'fix logging' tickets. ModuleResponder blacklist documentation is covered by the getting started docs ticket. Thanks for reporting this
